### DISCUSSION TITLE
Login issue

### DISCUSSION THREAD

**Discussion by gamerdj325** [SideStore/SideStore#470] [Q&A] [answered]
I can't login using my apple, it keeps loading over and over

**Comment by gamerdj325** [comment]
> I can't login using my apple id, it keeps loading over and over

**Reply by trevorwatkins1129** [reply]
I'm having the same promblem, have you had any luck?

**Comment by trevorwatkins1129** [comment]
I have found a solution: switch your anisette server to DrPudding.

**Comment by Spidy123222** [comment, marked as answer]
This was solved. Sidestore anisette was down for a bit.
